### PR TITLE
Updated Adobe Experience Cloud Solutions and Resource docs for 2.x support

### PR DESCRIPTION
**Pull request by swarna04**
ACP 2.x mobile SDK docs will contain the following

Experience Cloud Extensions
- Extension import & install instructions for 2.x
- Extension API reference
- Migration docs
- Release notes (2.x only)

Resources
- Privacy and GDPR
- Migrate to Swift
- FAQs

### CODE REVIEW

**Comment by emdobrin** [comment]
@swarna04 can you update SUMMARY.md for the removed files?

**Comment by swarna04** [comment]
Thanks @emdobrin. I've updated the summary page.

File: SUMMARY.md [hunk 1]
```diff
@@ -105,72 +104,28 @@
...
-  * [Mobile extension samples](resources/building-mobile-extensions/sample-extensions.md)
 * [Frequently asked questions](resources/frequently-asked-questions/README.md)
   * [Latest SDK Versions](https://aep-sdks.gitbook.io/docs/resources/upgrading-to-aep/current-sdk-versions)
 * [Privacy and GDPR](resources/privacy-and-gdpr.md)
-* [WeChat Mini Programs](resources/adobe-experience-platform-mini-programs-sdk/README.md)
-  * [Implementation](resources/adobe-experience-platform-mini-programs-sdk/implementation.md)
-  * [Implementation - 中文–简体](resources/adobe-experience-platform-mini-programs-sdk/implementation-zhong-wen-1.md)
-  * [Frequently Asked Questions](resources/adobe-experience-platform-mini-programs-sdk/faq.md)
 
 ## BETA
```
Comment (emdobrin) [inline SUMMARY.md:130]: should we remove this beta section too? we have a section for Assurance under Foundation extensions, I think we should keep any tutorials in the main branch only.

Comment (swarna04) [reply SUMMARY.md:130]: Yes we should, I wasn't sure if it was being done as part of one of the other PRs. I can make the updates here.

Comment (emdobrin) [reply SUMMARY.md:130]: 👍 Cool, not covered in my PRs at this point, so yup let's remove that too.

**Review by emdobrin — APPROVED** [review]

**Pull request merged**